extras/data/machinery/generate/EMC.py

```diff
@@ -0,0 +1,40 @@
+from .utils import load, obs_template
+
+
+def dump(src_path, target):
+    """Compute SLAC observables.
+
+    Parameters
+    ----------
+    src_path : str
+        input path
+
+    Returns
+    -------
+    dict
+        observables dictionary, corresponding to the runcard
+
+    """
+    del target
+
+    obs = obs_template.copy()
+    data = load(src_path, 0, ["-", "x", "Q2", "y"])
+    esf = [dict(x=d["x"], y=d["y"], Q2=d["Q2"]) for d in data]
+
+    obs["prDIS"] = "NC"
+    obs["observables"] = {"F2_total": esf}
+    obs["ProjectileDIS"] = "electron"
+
+    obs["TargetDIS"] = "proton"
+
+    return obs
+
+
+# renaming
+new_names = {
+    "emc_c_d": "EMC90_C_D",
```

Comment:
The names should be compatible to https://github.com/NNPDF/nnpdf/pull/1708 and I think more similar to the others ...

Reply:
This naming convention was actually decided in https://github.com/NNPDF/fktables/issues/34 to make the nuclear datasets (ratio) more explicit.

Reply:
I mean something like `EMC_NC_XXGEV_C_D_SIGMARED`